<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lords" name="NewTours_Login" tests="1" failures="1" timestamp="5 Nov 2019 11:35:01 GMT" time="16.232" errors="0">
  <testcase name="Login" time="1.219" classname="com.DataDriven.NewTours_Login">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.DataDriven.NewTours_Login.Login(NewTours_Login.java:48)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
</testsuite> <!-- NewTours_Login -->
